Packs/Azure-Enrichment-Remediation/Playbooks/playbook-Azure_-_User_Investigation.yml

```diff
@@ -0,0 +1,1387 @@
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "22"
+    scriptarguments:
+      key:
+        simple: AzureFailLoginMFACount
+      value:
+        complex:
+          root: AzureNumOfFailMFA.tables.rows.[0]
+          accessor: '[0]'
+          transformers:
+          - operator: SetIfEmpty
+            args:
+              applyIfEmpty: {}
+              defaultValue:
+                value:
+                  simple: "0"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 510,
+          "y": 1120
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "35":
+    id: "35"
+    taskid: 752b0d29-0d1f-4985-814f-72d96252984b
+    type: regular
+    task:
+      id: 752b0d29-0d1f-4985-814f-72d96252984b
+      version: -1
+      name: Set the number that the user was defined as a risky user
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.11/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "22"
+    scriptarguments:
+      key:
+        simple: AzureRiskyUserCount
+      value:
+        complex:
+          root: AzureRiskyUser.tables.rows.[0]
+          accessor: '[0]'
+          transformers:
+          - operator: SetIfEmpty
+            args:
+              applyIfEmpty: {}
+              defaultValue:
+                value:
+                  simple: "0"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -390,
+          "y": 760
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "1_22_#default#": 0.1
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1845,
+        "width": 2540,
+        "x": -810,
+        "y": 70
+      }
+    }
+  }
+inputs:
+- key: Username
+  value: {}
+  required: false
+  description: The username to investigate.
+  playbookInputQuery:
+- key: AzureSearchTime
+  value:
+    simple: ago(7d)
+  required: false
+  description: 'The Search Time for the Azure Log Analytics search query. Default value: ago(1d)'
```

Comment:
Is this the correct way to write the default value?

Reply:
yes